conformance/utils/http/http.go

```diff
@@ -229,3 +232,26 @@ func CompareRequest(cReq *roundtripper.CapturedRequest, cRes *roundtripper.Captu
 	}
 	return nil
 }
+
+// Get User-defined test case name or generate from expected response to a given request.
+func (er *ExpectedResponse) GetTestCaseName(i int) string {
+
+	/* If TestCase name is provided then use that or else generate one */
```

Comment:
Tiny nit: Would expect this to be a line comment (`//` instead of `/*`) per https://go.dev/doc/effective_go#commentary.

Reply:
Corrected.